oeps/oep-0052/decisions/003-edx.org-will-begin-with-saas-event-bus.rst

```diff
@@ -0,0 +1,37 @@
+edX.org Event Bus Deployment Choice
```

Comment:
Note: I still want an updated version of this documented somewhere, but I need to find the right home for it so it isn't blocked on the OEP. I'll either find a different repo, unless we are happy to land some `oep-0052/decisions` in advance of landing the actual OEP?